lib/msal-core/src/ServerRequestParameters.ts

```diff
@@ -137,10 +137,12 @@ export class ServerRequestParameters {
     str.push(`x-client-SKU=${this.xClientSku}`);
     str.push(`x-client-Ver=${this.xClientVer}`);
 
-    str.push("prompt=" + encodeURI(this.promptValue));
+    if (this.promptValue) {
+      str.push("prompt=" + encodeURI(this.promptValue));
```

Comment:
does this need an & after ?

Reply:
There is a generic API that joins all the strings with '&' following this function.